api/app/db/models/weather_models.py

```diff
@@ -184,7 +184,7 @@ class ModelRunPrediction(Base):
     # The date and time to which the prediction applies.
     prediction_timestamp = Column(TZTimeStamp, nullable=False, index=True)
     # The station code representing the location (aka weather station).
-    station_code = Column(Integer, nullable=True)
+    station_code = Column(Integer, nullable=False)
```

Comment:
I'm guessing this should have always been `False`, correct?

Reply:
Yep. I think I originally had it set to True as a column was being added to an existing table so there was the possibility of null values and providing a default didn't make sense. In the end I deleted the contents of the table in the migration and had the migration set nullable=False. This change brings the code into alignment with the current schema of the db.